### DISCUSSION TITLE
dual pane + preview, esp. in terminator

### DISCUSSION THREAD

**Discussion by rkoe** [jarun/nnn#1923] [Q&A] [answered]
I'm new to nnn, it looks really nice, and would like to use it like in the video on the top of the README.

1. Where can I find the configuration, used in this video? It would be a nice starting-point.
2. Does this also work with the terminal-program "Terminator"?
    - Dual pane: The wiki says, that dual pane also works with Terminator (https://github.com/jarun/nnn/wiki/Concepts#dual-pane), but I don't know how to set this up. Can you give me a hint, or even an example?
    - Preview: Is preview (preview-tabbed or preview-tui), with the preview shown in the right pane, possible in Terminator?
      If not directly, maybe with Terminator+tmux?

Thanks

**Comment by N-R-K** [comment, marked as answer]
Dual pane is just opening two different nnn instance in two splits. There's nothing to setup.

> If not directly, maybe with Terminator+tmux?

Yes, tmux is supported. See [documentation in the plugin](https://github.com/jarun/nnn/blob/af9bfd370faa90966cc75a93feceec7cb5bb2321/plugins/preview-tui#L3-L98) for a list of dependencies and usage instruction.
